framework/components/ecl-forms/ecl-forms-textareas/ecl-forms-textareas.twig

```diff
@@ -1,15 +1,39 @@
-<div class="form-group">
-  <label for="field-ID-1">Field label</label>
-  <p class="help-block">This is some placeholder help text.</p>
-  <textarea name="field-name" id="field-ID-1" rows="4" class="form-textarea form-control"></textarea>
-</div>
-
-<div class="form-group">
-  <label for="field-ID-2">Field label - error</label>
-  <p class="help-block">This is some placeholder help text.</p>
-  <textarea name="field-name" id="field-ID-2" rows="4" class="form-control form-textarea error"></textarea>
-  <div class="messages error">
-    <h3><span class="ecl-sr-only">Error message:</span>Some feedback title</h3>
-    <p>Some feedback message</p>
-  </div>
-</div>
+{% set id = id|default('default-id') %}
```

Comment:
Please add a comment describing the accepted parameters (as discussed this morning) :)